### PR TITLE
readme: update readme, clarify some guidelines and improve example

### LINKED ISSUE #2: Add Contribution Guideline
We could adapt the global gluon Contribution Guideline and add some stuff, for example:

- At the moment, the intention of a package can only be guessed by reading the description in the `Makefile` or if not given only guess by name and reading the code.
    We should add a central place in te Contribution guideline where to put a thorough description.

### PR DESCRIPTION
**Pull request by maurerle**
I update the readme and clarified some information on how to contribute and which things have to be respected.

This makes sure that
* contributions have the same pattern
* commits relate to a given package
* makes sure that a license is given
* introduces the community-independant ff- slug
* clarifies some guidelines of the community-packages workflow

fixes #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,41 +13,54 @@ GLUON_SITE_FEEDS='community'
 
 PACKAGES_COMMUNITY_REPO=https://github.com/freifunk-gluon/community-packages.git
 PACKAGES_COMMUNITY_COMMIT=<COMMIT-HASH>
+PACKAGES_COMMUNITY_BRANCH=master
 ```
 
-## Commit guidelines
+If you leave out the `_BRANCH`, the repository default branch is used - your commit has to be on the specified branch.
+
+You should always test and check changes before deploying a new commit, as many people of the community have commit rights and there is currently no test or release process defined.
```
Comment (grische) [inline README.md:21]: Shouldn't this be part of the contribution guidelines? If you meant to add this for users, maybe replace "before deploying a new commit" with "before using a new commit of the community-packages in your modules" or something like that.

File: README.md [hunk 2]
```diff
@@ -13,41 +13,54 @@ GLUON_SITE_FEEDS='community'
...
+
+## Contribution guidelines
+
+### Commit guidelines
 
 * The master branch is protected against the use of `git push --force`.
-* Use Pull Requests if you are uncertain about your patches.
+* Use Pull Requests if you are uncertain about your patches - this is generally a good practice to get feedback from others.
```
Comment (grische) [inline README.md:28]: If someone is unsure, maybe we can introduce adding an `[WIP]` or `[RFC]` tag in front of the PR name?

File: README.md [hunk 3]
```diff
@@ -13,41 +13,54 @@ GLUON_SITE_FEEDS='community'
...
-### PKG_LICENSE
+It is also possible to add packages with the `ff` slug to generalize the ownership or add packages without a tying maintenance to a specific community.
+
+#### PKG_LICENSE
 
 The PKG License should be defined as a SPDX ID. See the SPDX FAQ for more details.
 
 [spdx.dev/learn/handling-license-info](https://spdx.dev/learn/handling-license-info/#how)
 
+It is mandatory for new packages to include a license in the Makefile
```
Comment (grische) [inline README.md:61]: Maybe change the `FantasyLicense` in the example below to a proper license to make it easier understanding the SPDX headers.

**Review by grische — COMMENTED** [review]
some small comments

File: README.md (hunk 2 above)
Comment (maurerle) [reply README.md:28]: imho using draft PRs is enough.
I also had the experience of forgetting to change the PR name leading to some `WIP: bla` being merged on github in some project - but if you insist I can add something to reflect this?

Comment (grische) [reply README.md:28]: All good, just an idea.

File: README.md (hunk 1 above)
Comment (maurerle) [reply README.md]: I meant this for users - adjusted the comment accordingly

File: README.md (hunk 2 above)
Comment (grische) [reply README.md]: Thanks

**Review by grische — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -13,41 +13,73 @@ GLUON_SITE_FEEDS='community'
 
 PACKAGES_COMMUNITY_REPO=https://github.com/freifunk-gluon/community-packages.git
 PACKAGES_COMMUNITY_COMMIT=<COMMIT-HASH>
+PACKAGES_COMMUNITY_BRANCH=master
 ```
 
-## Commit guidelines
+Your commit has to be on the branch specified in `PACKAGES_COMMUNITY_BRANCH`.
+If you leave out the `PACKAGES_COMMUNITY_BRANCH`, the repository default branch (master) is used.
```
Comment (grische) [inline README.md:20]: That's not correct anymore. The default branch selected on GitHub is being used, not master.

**Review by grische — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (maurerle) [reply README.md:20]: sure, whatever the owner of the repository sets as repository default branch - but this is the master branch (currently).
If we change this, we need to adjust the docs too, anyway - as this is written before too?

Comment (maurerle) [reply README.md:20]: I meant do be more clarifying. I can remove the brackets too

Comment (grische) [reply README.md:20]: I think removing the brackets makes most sense.

**Review by grische — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (grische) [reply README.md]: Thanks!

**Review by mkg20001 — APPROVED** [review]

**Pull request merged**